frontend/src/services/normService.ts

```diff
@@ -51,6 +53,38 @@ export async function getNormHtmlByEli(
   })
 }
 
+/**
+ * Load the rendered html version of a norm from the api using useFetch
+ *
+ * @param eli Eli of the norm
+ * @param showMetadata Whether to include metadata in the rendered HTML
+ * @param at Date indicating which modifications should be applied before the HTML gets rendered and returned
+ */
+export function useGetNormHtmlByEli(
```

Comment:
`useNormHtml`?

Reply:
oh that is what the existing composable is already called. Maybe we should atleast merge the new composable (`useNormHtmlByEli` and this one)